webpack.config.js

```diff
@@ -11,6 +11,7 @@ const config = {
     path: path.resolve(__dirname, 'dist'),
     filename: 'index.js',
     libraryTarget: 'umd',
+    hashFunction: 'xxhash64',
```

Comment:
who's this fella

Reply:
Just copying Joe Kent's work https://github.com/ArcadiaPower/connect-react/pull/54

Reply:
Looks like it's because of this https://github.com/webpack/webpack/issues/14532